src/main/resources/application.conf

```diff
@@ -14,6 +14,8 @@ etc-client {
 
   network {
 
+    protocol-version = "1"
```

Comment:
[Here](https://github.com/ethereum/wiki/wiki/JSON-RPC#net_version) it says: 

> Returns the current network protocol version.

Parity is returning me `1` but take a look at this: https://github.com/ethereum/EIPs/issues/611 :thinking: 

> go-ethereum and Parity too currently returns the eth network ID, which is probably the worse possible interpretation of this field based on its name and description.